deps/config/types.go

```diff
@@ -70,6 +70,23 @@ func DefaultCurioConfig() *CurioConfig {
 				AlertManagerURL: "http://localhost:9093/api/v2/alerts",
 			},
 		},
+		Market: MarketConfig{
+			StorageMarketConfig: StorageMarketConfig{
+				PieceLocator: []PieceLocatorConfig{},
+				Indexing: IndexingConfig{
+					InsertConcurrency: 8,
+					InsertBatchSize:   15000,
+				},
+				MK12: MK12Config{
+					Libp2p:                    []Libp2pConfig{},
+					PublishMsgPeriod:          Duration(30 * time.Minute),
+					MaxDealsPerPublishMsg:     8,
+					MaxPublishDealsFee:        types.MustParseFIL("5 FIL"),
```

Comment:
Per message makes more sense in the way that It is just good default. Easy to calculate based on gas trends. Per deal would need users to calculate per deal based on current gas fee in a more complicated manner.

Reply:
Economically though as an SP it would make much more sense for me to say "I want to pay X for each sector", X for 1~max sectors is much less useful knob imho.

We can ship with some reasonable default and say e.g. 'this amount was ~300pFIL basefee on [date]'